ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -102,26 +117,32 @@ const AssetListControlBar = ({ showTokensLinks }: AssetListControlBarProps) => {
           </ButtonBase>
         )}
 
-        <ButtonBase
-          data-testid="sort-by-popover-toggle"
-          className="asset-list-control-bar__button"
-          onClick={toggleTokenSortPopover}
-          size={ButtonBaseSize.Sm}
-          endIconName={IconName.ArrowDown}
-          backgroundColor={
-            isTokenSortPopoverOpen
-              ? BackgroundColor.backgroundPressed
-              : BackgroundColor.backgroundDefault
-          }
-          borderColor={BorderColor.borderMuted}
-          borderStyle={BorderStyle.solid}
-          color={TextColor.textDefault}
-          marginRight={isFullScreen ? 2 : null}
+        <Box
+          className="asset-list-control-bar__buttons"
+          display={Display.Flex}
+          justifyContent={JustifyContent.flexEnd}
         >
-          {t('sortBy')}
-        </ButtonBase>
+          <Tooltip title={t('sortBy')} position="bottom" distance={20}>
+            <ButtonBase
+              data-testid="sort-by-popover-toggle"
+              className="asset-list-control-bar__button"
+              onClick={toggleTokenSortPopover}
+              size={ButtonBaseSize.Sm}
+              endIconName={IconName.SwapVertical}
+              backgroundColor={
+                isTokenSortPopoverOpen
+                  ? BackgroundColor.backgroundPressed
+                  : BackgroundColor.backgroundDefault
+              }
+              color={TextColor.textDefault}
+              marginRight={isFullScreen ? 2 : null}
+            />
```

Comment:
Should this use `ButtonIcon`?

Reply:
Yeah we probably should. I'll make this change ty for pointing this out.